### DISCUSSION TITLE
Upgraded helm-controller to v0.32.2 and seeing this error on all the helm releases: failed to verify artifact: invalid checksum digest format

### DISCUSSION THREAD

**Discussion by soumyabk** [fluxcd/flux2#3864] [Q&A]
We are using our own internal helm repository. We were using helmcontroller version 0.26.0, but we wanted to enable the drift detection feature so tried upgrading to 0.32.2 and all our helm releases are failing with this error: 
`failed to verify artifact: invalid checksum digest format`

**Comment by soumyabk** [comment]
Earlier tried upgrading to v0.31.2 but ran into this error discussed here - https://github.com/fluxcd/helm-controller/issues/669

**Comment by stefanprodan** [comment]
You need to upgrade Flux to RC.2 along with all controllers and API versions in Git, the checksums will be recomputed after that

**Comment by soumyabk** [comment]
it wouldn't be possible for us to upgrade immediately. Is it possible for us to use v0.31.2 and handle the case where when a new helmrelease is deployed it doesn't fail with `runtime error: invalid memory address or nil pointer dereference)`
It seems to have worked with helmreleases that were already present.

**Comment by soumyabk** [comment]
upgraded all controller to rc.2
